### ISSUE TITLE
ping fail

### ISSUE THREAD

**Issue by justinjie** [atc-/docker-registry-ui#152]
hi:
I start  as:
```
 sudo docker run -p 8080:8080 atcol/docker-registry-ui
```
when I create a private registry in the web, it shows ping fail.
then the result as: 
```
2016-01-14 09:01:45,697 [localhost-startStop-1] INFO  conf.BootStrap  - Checking for registries in system env
2016-01-14 09:01:45,708 [localhost-startStop-1] INFO  conf.BootStrap  - Found registry http://:5000/v1. Creating...
2016-01-14 09:01:45,712 [localhost-startStop-1] ERROR conf.BootStrap  - Couldn't parse valid registry URL from http://:5000/v1
Jan 14, 2016 9:01:45 AM org.apache.coyote.AbstractProtocol start
INFO: Starting ProtocolHandler ["http-bio-8080"]
Jan 14, 2016 9:01:45 AM org.apache.catalina.startup.Catalina start
INFO: Server startup in 26896 ms
2016-01-14 09:02:40,630 [http-bio-8080-exec-10] INFO  web.RepositoryService  - Ping failed: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost:5000 refused
2016-01-14 09:02:58,656 [http-bio-8080-exec-10] ERROR web.RepositoryService  - Failed to ping http://192.168.2.159:5000/v1/_ping: 404 : Not Found
2016-01-14 09:03:08,004 [http-bio-8080-exec-9] ERROR web.RepositoryService  - Failed to ping http://192.168.56.100:5000/v1/_ping: 404 : Not Found
```
how can I solve this problem?
thank you very much

**Comment by LouisKottmann** [comment]
It is trying to reach `http://:5000/v1`, pretty sure that's not the address of your registry ;)

**Comment by justinjie** [comment]
Thank you very much. But I also try to other address, such as 
```
172.17.0.3:5000
localhost
```
could you tell me  what is the address of my registry @LouisKottmann

**Comment by LouisKottmann** [comment]
It is wherever you put it, I don't believe docker-registry-ui has it built-in

**Comment by justinjie** [comment]
thank you very much, I think I have solved the problem, as the issue  #148. Because the registry I used is V2 , and the images docker-registry-ui not support it. There is a new images to support it, which can be found in the issue  #148.
